NataliiaZinchenko/HW6/nzinchenko_6.2.py

```diff
@@ -0,0 +1,8 @@
+entered_login = input("Enter your login: ").lower()
```

Comment:
in your version of code operator "in" will consider not only the login "first" as correct, but also all combinations of words that contain the word "first"

look at this implementation
user_name = input('Hello, please input your Log in:')

while user_name != 'First':
    user_name = input('Error: wrong username, please try one more time. Username:')
else:
    print('Greeting. Access granted!!!', user_name)